org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/PluginImages.java

```diff
@@ -47,8 +47,7 @@
     CANCEL_BUILD_EXECUTION(ImmutableMap.of(ImageState.ENABLED, "icons/full/elcl16/cancel_build_execution.png", ImageState.DISABLED, "icons/full/dlcl16/cancel_build_execution.png")),
     RERUN_BUILD(ImmutableMap.of(ImageState.ENABLED, "icons/full/elcl16/rerun.png", ImageState.DISABLED, "icons/full/dlcl16/rerun.png")),
     RERUN_FAILED_TESTS(ImmutableMap.of(ImageState.ENABLED, "icons/full/elcl16/rerun_failed_tests.png", ImageState.DISABLED, "icons/full/dlcl16/rerun_failed_tests.png")),
-    EXPAND_NODE(ImmutableMap.of(ImageState.ENABLED, "icons/full/elcl16/expand_node.png")),
-    COLLAPSE_NODE(ImmutableMap.of(ImageState.ENABLED, "icons/full/elcl16/collapse_node.png")),
+    EXPAND_ALL(ImmutableMap.of(ImageState.ENABLED, "icons/full/elcl16/expand_all.png")),
```

Comment:
Out of curiosity, where does this icon come from? I guess only the collapse all icon is available from the shared images.

Reply:
if you do some search(Ctrl+H) in eclipse, you will find the result view has a expand_all icon , and yeah , it is not in shared images, it is in search related plugins , I just copy it from source.